fritz

```diff
@@ -123,16 +129,24 @@ def run(args):
             raise RuntimeError("Failed to update fritz's submodules")
 
     if args.init:
+        # check config
         check_config(cfg='fritz.defaults.yaml', yes=args.yes)
 
-    if args.init:
+        # add Fritz-specific K and SP extensions:
+        copy_tree('extensions/skyportal/', 'skyportal/')
+        # copy over the additional requirements:
+        subprocess.run(["cp", ".requirements/ext.txt", f"skyportal/requirements.fritz.txt"], check=True)
+
         # Build skyportal's images
         p = subprocess.run(["make", "docker-local"], cwd="skyportal")
         if p.returncode != 0:
             raise RuntimeError("Failed to build skyportal's docker images")
 
     # start up skyportal
-    p = subprocess.run(["docker-compose", "up", "-d"], cwd="skyportal")
+    # create kowalski_kowalski network if it does not exist yet
+    p = subprocess.run(["docker", "network", 'create', "kowalski_kowalski"])
```

Comment:
Re: our conversation on slack, could be good to append a SHA to `kowalski_kowalski` to avoid collisions

Reply:
Let's do this in a separate PR? Note for our future selves: the SHA should be set in `fritz.yaml` and propagated to the appropriate places. Filed an issue #14.